packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -52,14 +75,18 @@ export const computeRoundTiming = (params, baseTime) => {
   const steps = TimeMath.divideRelRel(targetDuration, clockStep);
   const duration = TimeMath.multiplyRelNat(clockStep, steps);
 
-  steps > 0n ||
-    Fail`clockStep ${clockStep} too long for auction duration ${duration}`;
+  if (steps <= 0n) {
```

Comment:
this is really just checking for `0n` right? since lines before this ensure it's non-negative. worth a comment

Reply:
yes.  I carefully reversed the previous tests, and double checked I hadn't missed any. This only needs to check for 0.